crates/stackable-operator/src/commons/authentication/mod.rs

```diff
@@ -77,6 +78,9 @@ pub enum AuthenticationClassProvider {
     /// The [TLS provider](DOCS_BASE_URL_PLACEHOLDER/concepts/authentication#_tls).
     /// The TLS AuthenticationClass is used when users should authenticate themselves with a TLS certificate.
     Tls(tls::AuthenticationProvider),
+
+    /// The Kerberos provider is used for Kerberos authentication and defines the secret used for generating keytabs.
```

Comment:
nit: I think the addition might be a bit misleading.
```suggestion
  /// The Kerberos AuthenticationClass is used when users should authenticate themselves via Kerberos.
```